ui/app/templates/components/job-editor/alert.hbs

```diff
@@ -0,0 +1,29 @@
+  <div class="job-editor-alerts">
+    {{#if @data.error}}
+      <Hds::Alert @type="inline" @color="critical" data-test-error={{@data.error.type}} as |A|>
+          <A.Title data-test-error-title>{{conditionally-capitalize @data.error.type true}}</A.Title>
+          <A.Description data-test-error-message>{{@data.error.message}}</A.Description>
+      </Hds::Alert>
+    {{/if}}
+    {{#if (and (eq @data.stage "read") @data.hasVariables (eq @data.view "job-spec"))}}
+      {{#if this.shouldShowAlert}}
+        <Hds::Alert @type="inline" @onDismiss={{this.dismissAlert}} data-test-variable-notification as |A|>
+          <A.Title>All HCL variable values Not Shown</A.Title>
+          <A.Description>All HCL variable values provided to this job version may not be shown. Ensure HCL variable values are properly set when submitting the job.</A.Description>
+        </Hds::Alert>
+      {{/if}}
```

Comment:
Is this just always on?  "Ensure that..." in an alert reads to me like I've made some kind of mistake that I could correct in the future, in order to not see it.

Noting this because it seems like `shouldShowAlert` is set to true, not dependent on anything else other than a user dismissing it. Is there something I'm missing here?

<img width="894" alt="image" src="https://user-images.githubusercontent.com/713991/236558028-acec8ab2-fbf4-4482-8293-4701733d184d.png">

What impact are we hoping to have with the user in showing them this?

Reply:
A second thought, but even if we **did** intend to show this to a user on all jobs, could we possibly make it localStorage dismissable? If the goal is to show it to all users, but it isn't actually impacted by the job in question, could we make it a `@localStorageProperty` so that once dismissed, it stays dismissed (in the user's current browser)